composer.lock

```diff
@@ -1,7160 +0,0 @@
-{
```

Comment:
`composer.lock` must be preserved

Reply:
Somehow I missed this one:)

Reply:
Did you generate this with the lowest supported platform? 🤔